core/storage/exploration/gae_models.py

```diff
@@ -239,59 +239,6 @@ def has_reference_to_user_id(cls, unused_user_id):
         return False
 
 
-class ExplorationMathRichTextInfoModel(base_models.BaseModel):
```

Comment:
Maybe delete this one in a future PR (file an issue to track) that is merged only once we're sure that no such models exist on any of the servers.